app/frontend/src/lib/ui/components/filterGroup.js

```diff
@@ -6,6 +6,9 @@ export const CHECKBOX_CLASS_SELECTOR = 'govuk-checkboxes__input';
 window.addEventListener('DOMContentLoaded', () => init('filter-group__container', 'moj-filter__tag', 'clear-filters-button', 'close-all-groups'));
 
 export const init = (groupContainerSelector, removeButtonSelector, clearButtonSelector, closeButtonSelector) => {
+  // Prevent automatic submit on service start page
+  if (window.location.pathname === '/') { return; }
```

Comment:
i think there is a better solution that this that would make things a bit more testable and a configuration of the component. i propose putting a data attribute on the form in question of data-auto-submit="true" or something similar and checking for this is the logic of the code somehow. i think the code might need a bit of a re-gig to accomodate this, but not too much upheaval and happy to help. ill create a PR to your feature branch